packages/@ember/-internals/routing/lib/system/router.ts

```diff
@@ -802,7 +802,7 @@ class EmberRouter extends EmberObject.extend(Evented) implements Evented {
       this._toplevelView = null;
     }
 
-    this._super(...arguments);
+    super.willDestroy();
```

Comment:
Per errors from tslint, we need to drop `...arguments` here because the underlying base method for `willDestroy` is already known to accept 0 arguments.